### DISCUSSION TITLE
Expected publication in eNotices2

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#852] [Q&A] [answered]
Hello,

In eNotices2 we see that the "Expected publication date" column shows a date that we did not send in the notices.

In the preview environment in notices that we sent today, March 6, it appears as "Expected publication date" February 28.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/a7f1d64a-ebbc-4418-8062-37ebf14dc1a6)

How is the value of that date generated?

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
The issue you are reporting is documented on our Preview page, which we update regularly for known issues:
[The following issue has been identified as of 27 February 2024 in Preview:](https://docs.ted.europa.eu/home/eforms/preview/index.html#_enotices2:~:text=The%20following%20issue,expected%20publication%20date%E2%80%9D.)
[Notices submitted to Preview eNotices2 as of 27 February 2024 do not acquire status “published” due to an error in the export process. We are currently working to fix this issue, which may also affect the “expected publication date”.](https://docs.ted.europa.eu/home/eforms/preview/index.html#_enotices2:~:text=The%20following%20issue,expected%20publication%20date%E2%80%9D.)
In the future, please contact our Helpdesk for non-SDK related questions.
